
FAIL XMLHttpRequest: abort() after a timeout should not fire "abort" event assert_true: expected true got false(stack: assert@http://localhost:8800/resources/testharness.js:1738:42
assert_true@http://localhost:8800/resources/testharness.js:299:15
http://localhost:8800/XMLHttpRequest/abort-after-timeout.htm:29:24
step@http://localhost:8800/resources/testharness.js:798:30
http://localhost:8800/resources/testharness.js:827:40)

